libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/junction/api/MapboxJunctionApi.kt

```diff
@@ -23,6 +24,7 @@ import kotlinx.coroutines.launch
  * Mapbox Junction Api allows you to generate junction for select maneuvers.
  * @property accessToken String
  */
+@UiThread
```

Comment:
Why? We don't have any mutable state here and we proxy to main thread ourselves when we make a request. I don't think this is a valid restriction: it can be called from any thread.